zeek-aux/zeek-aux.spec

```diff
@@ -8,7 +8,7 @@ License:        BSD
 URL:            https://github.com/zeek/zeek-aux
 Source0:        https://www.zeek.org/downloads/%{name}-%{version}.tar.gz
 
-Provides:       bro-aux
+Provides:       zeek-aux
```

Comment:
`zeek-aux` is already the name of the package, so it already provides that. Specifying `bro-aux` implies that it is sufficient to replace the `bro-aux` package